extensions/amp-autocomplete/0.1/amp-autocomplete.js

```diff
@@ -394,7 +445,7 @@ export class AmpAutocomplete extends AMP.BaseElement {
       if (resultsShowing) {
         this.activeIndex_ = mod(index, this.container_.children.length);
         newActiveElement = this.container_.children[this.activeIndex_];
-        newValue = newActiveElement.textContent;
+        newValue = newActiveElement.getAttribute('value');
```

Comment:
If this is a change to the API, make sure you call it out in a section of your doc and update the public documentation.

Reply:
I added the section [Supporting plain and rich content](https://docs.google.com/document/d/1OOPnKgShbh7BN2D24u7bxa57z2spAiBA8n9o0L5V3qA/edit#heading=h.kd3r11jhzfrz) to the design doc to explain the reasoning and provide examples. Let me know if this change is reasonable to you. I'll also add some lines to the md file.